geometry/create_noise_file_chargePreAmp_theta.py

```diff
@@ -55,7 +55,7 @@ def get_ref_charge(SF, E_dep = 1 * pow(10, 6)): #E_dep en eV, choose 1 MeV
 
 # Sampling fraction in each layer
 # numbers updated for latest model with 1536 modules
-SFfcc = [0.36599110182660616] * 1 + [0.1366222373338866] * 1 + [0.1452035173747207] * 1 + [0.1504319190969367] * 1 + [0.15512713637727382] * 1 + [0.1592916726494782] * 1 + [0.16363478857307595] * 1 + [0.1674697333180323] * 1 + [0.16998205747422343] * 1 + [0.1739146363733975] * 1 + [0.17624609543603845] * 1 + [0.1768613530850488] * 1
+SFfcc = [0.3864252122990472] * 1 + [0.13597644835735828] * 1 + [0.14520427829645913] * 1 + [0.1510076084632846] * 1 + [0.1552347580991012] * 1 + [0.159694330729184] * 1 + [0.1632954482794191] * 1 + [0.16720711037339814] * 1 + [0.17047749048884808] * 1 + [0.17461698117974286] * 1 + [0.1798984163980135] * 1 + [0.17920355117405806] * 1
```

Comment:
```suggestion
SFfcc = [0.36599110182660616] * 1 + [0.1366222373338866] * 1 + [0.1452035173747207] * 1 + [0.1504319190969367] * 1 + [0.15512713637727382] * 1 + [0.1592916726494782] * 1 + [0.16363478857307595] * 1 + [0.1674697333180323] * 1 + [0.16998205747422343] * 1 + [0.1739146363733975] * 1 + [0.17624609543603845] * 1 + [0.1768613530850488] * 1,
```

Reply:
I mean, these sampling fractions should be the one from the theta based geometry no?

Reply:
the numbers I put in all scripts are the latest ones I got with allegro_o1_v02 and the theta-based readout

Reply:
the numbers are quite close but there are some little differences I assume they are related to some changes that took place between the two calculations for instance this one https://github.com/key4hep/k4geo/commit/953441c383999da140c901fd305ab51c1c34e7fa